core/templatetags/element_by_counter.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+register = template.Library()
+
+@register.simple_tag
+def element_by_counter(data, counter, divider):
```

Comment:
Could you add tests to this template tag? Could be in separate PR as we don't want to block this to be merged.

Reply:
Will work on this. Thanks